REQUIRE

```diff
@@ -3,3 +3,6 @@ MathProgBase
 PyPlot
 Clp
```

Comment:
Is there a special interest to use Clp more thant GLPK ? I do not defend one more than the other but I think the question should be raise